src/components/pages/ViewAllMeetings/VAMC.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState, useEffect } from 'react';
+import 'antd/dist/antd.css';
+import { Calendar, Modal, Badge, Button } from 'antd';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
+
+const initialValues = [
+  {
+    meeting_start_date: '01/04/2022',
+    type: 'warning',
+    content: 'This is warning event.',
+    last_name: 'Test information 1',
+    meeting_id: 1,
+    meeting_topic: 'testing models',
+  },
+  {
+    meeting_start_date: '04/04/2022',
+    type: 'success',
+    content: 'This is usual event.',
+    last_name: 'Test information 2',
+    meeting_id: 2,
+    meeting_topic: 'product review',
+  },
+  {
+    meeting_start_date: '06/04/2022',
+    type: 'error',
+    content: 'This is error event 1.',
+    last_name: 'Test information 3',
+    meeting_id: 3,
+    meeting_topic: 'other stuff',
+  },
+  {
+    meeting_start_date: '12/04/2022',
+    type: 'error',
+    content: 'This is error event 2.',
+    last_name: 'Test information 4',
+    meeting_id: 4,
+    meeting_topic: 'even more other stuff',
+  },
+  {
+    meeting_start_date: '15/04/2022',
+    type: 'error',
+    content: 'This is error event 3.',
+    last_name: 'Test information 5',
+    meeting_id: 5,
+    meeting_topic: 'Oh, some new other stuff now.',
+  },
+  {
+    meeting_start_date: '18/04/2022',
+    type: 'success',
+    content: 'This is usual event1.',
+    last_name: 'Test information 6',
+    meeting_id: 6,
+    meeting_topic: 'That was nice',
+  },
+  {
+    meeting_start_date: '27/04/2022',
+    type: 'success',
+    content: 'This is usual event2.',
+    last_name: 'Test information 7',
+    meeting_id: 7,
+    meeting_topic: 'other stuff. Again',
+  },
+];
+
+function VAMC(props) {
+  const { meetingData } = props;
+
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [isScheduleModalVisible, setIsScheduleModalVisible] = useState(false);
+  const [event, setEvent] = useState(null);
+  const [eventsArr, setEventsArr] = useState(initialValues);
+
+  // useEffect(() => {
```

Comment:
why is this useEffect commented out? Should it just be removed?

Reply:
We will be using it later. Just need to get the data correct.